terraform/api/aws/variables.tf

```diff
@@ -32,6 +32,10 @@ variable "name" {
   type = string
 }
 
+variable "rack_name" {
+  type = string
+}
```

Comment:
Since name is used by the k8s cluster and can't be changed, this variable will be responsible to have the current rack name.